istio-2-mixin/dashboards.libsonnet

```diff
@@ -0,0 +1,164 @@
+local g = import './g.libsonnet';
+local logslib = import 'logs-lib/logs/main.libsonnet';
+{
+  local root = self,
+  new(this):
+    local prefix = this.config.dashboardNamePrefix;
+    local links = this.grafana.links;
+    local tags = this.config.dashboardTags;
+    local uid = g.util.string.slugify(this.config.uid);
+    local vars = this.grafana.variables;
+    local annotations = this.grafana.annotations;
+    local refresh = this.config.dashboardRefresh;
+    local period = this.config.dashboardPeriod;
+    local timezone = this.config.dashboardTimezone;
+    local panels = this.grafana.panels;
+    local stat = g.panel.stat;
+    {
+      overview:
```

Comment:
Is there a way to give the output dashboard a name here? Maybe adding in a dashboardUid so that generated dashboards have the correct name and file type json as an output when running the `make` command.

Reply:
You can update this by using a string `'istio-service-overview.json'` or whatever the correct name for the dashboard you want in the dashboard.libsonnet in place of the `overview` variable.

Reply:
Actually, I am not sure if this is a solution if you have multiple dashboards that need to reference dashboard variables like in your case.